Lua/Plugins/c.lua

```diff
@@ -0,0 +1,138 @@
+-- Highlight Keywords
+highlight("auto", "reserved")
+highlight("break", "reserved")
+highlight("case", "reserved")
+highlight("char", "reserved")
+highlight("const", "reserved")
+highlight("continue", "reserved")
+highlight("default", "reserved")
+highlight("do", "reserved")
+highlight("double", "reserved")
+highlight("else", "reserved")
+highlight("enum", "reserved")
+highlight("extern", "reserved")
+highlight("float", "reserved")
+highlight("for", "reserved")
+highlight("goto", "reserved")
+highlight("if", "reserved")
+highlight("inline", "reserved")
+highlight("int", "reserved")
+highlight("long", "reserved")
+highlight("register", "reserved")
+highlight("restrict", "reserved")
+highlight("return", "reserved")
+highlight("short", "reserved")
+highlight("signed", "reserved")
+highlight("sizeof", "reserved")
+highlight("static", "reserved")
+highlight("struct", "reserved")
+highlight("switch", "reserved")
+highlight("typedef", "reserved")
+highlight("union", "reserved")
+highlight("unsigned", "reserved")
+highlight("void", "reserved")
+highlight("volatile", "reserved")
+highlight("while", "reserved")
+highlight("_Alignas", "reserved")
+highlight("_Alignof", "reserved")
+highlight("_Atomic", "reserved")
+highlight("_Bool", "reserved")
+highlight("_Complex", "reserved")
+highlight("_Generic", "reserved")
+highlight("_Imaginary", "reserved")
+highlight("_Noreturn", "reserved")
+highlight("_Static_assert", "reserved")
+highlight("_Thread_local", "reserved")
+
+-- Arithmetic Operators
+highlight("+", "operator") 
+highlight("-", "operator")
+highlight("*", "operator")
+highlight("/", "operator")
+highlight("%", "operator")
+highlight("++", "operator")
+highlight("--", "operator")
+
+-- Assignment Operators
+highlight("=", "operator")
+highlight("+=", "operator")
+highlight("-=", "operator") 
+highlight("*=", "operator")
+highlight("/=", "operator")
+highlight("%=", "operator")
+highlight("&=", "operator")
+highlight("^=", "operator")
+highlight("|=", "operator")
+highlight("<<=", "operator")
+highlight(">>=", "operator")
+
+-- Comparison Operators
+highlight("==", "operator")
+highlight("!=", "operator")
+highlight(">", "operator")
+highlight("<", "operator")
+highlight(">=", "operator")
+highlight("<=", "operator")
+
+-- Logical Operators 
+highlight("&&", "operator")
+highlight("||", "operator")
+highlight("!", "operator")
+
+-- Bitwise Operators
+highlight("&", "operator")
+highlight("|", "operator")
+highlight("^", "operator")
+highlight("~", "operator")
+highlight("<<", "operator")
+highlight(">>", "operator")
+
+-- Special Characters
+highlight("{", "binary")
+highlight("}", "binary")
+highlight("[", "binary")
+highlight("]", "binary")
+highlight("(", "binary")
+highlight(")", "binary") 
+highlight(";", "binary")
+highlight(",", "binary")
+
+-- Strings 
+highlight_region("\"", "\"", "string")
+highlight_region("'", "'", "string")
+
+-- Comments
+add_comment("//", "", "comments", true)
+add_comment("/*", "*/", "comments", false)
```

Comment:
```suggestion
highlight_region("//", "", "comments", true)
highlight_region("/*", "*/", "comments", false)
```
`add_comment` is used in the <kbd>Control</kbd> + <kbd>L</kbd> menu, not the actual language comments.
Also, you need to add at least 3 comments (`add_comment("comment")`) before this can be merged.